scripts/testing/.gitignore

```diff
@@ -0,0 +1,4 @@
+nohup.out
+Vera.ttf
+*.csv.gz.*
```

Comment:
Same comment here about where to store things - let's update this so that the output files end up in `tmp/spectrum`, and create that directory using Ansible (see `ansible/roles/common/tasks/tools.py` for how the other temp directories are created).

Reply:
I now suggest working from home dir to prevent an unclean repo state.
Also, it deletes the intermediate files after generating output.

If you want it in the website, we could copy/move the resulting file somewhere meaningful.